pkg/local_object_storage/blobstor/fstree/fstree_write_linux.go

```diff
@@ -37,12 +38,15 @@ func (t *FSTree) writeFile(p string, data []byte) error {
 				err = nil
 			}
 		} else {
-			err = errors.New("incomplete write")
+			err = errors.New("incomplete unix write")
```

Comment:
Not worth the trouble.

Reply:
what `write`? this exact one

Reply:
i think, a user should understand if he use unix or not

Reply:
but more (obvious) details are never bad to me